packages/frontend/src/utils/formatters/shortenHexString.ts

```diff
@@ -1,4 +1,4 @@
 import { Hex } from 'viem'
 
 export const shortenHexString = (hexString: Hex, end = 4) =>
-  `${hexString.substring(0, 6)}......${hexString.substring(hexString.length - end)}`
+  `${hexString.substring(0, 4)}....${hexString.substring(hexString.length - end)}`
```

Comment:
Let's change the function to accept optional start argument, that's 6 by default and use it instead of 4. On the bids short list on desktop view we should show six chars instead of 4